docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -13,7 +13,7 @@ A note on development images
 Both PELUX images are available as `-dev` variants, which include some extra
 development and debugging tools. For some platforms, those images are also
 available as `-update` to generate update artifacts for the :ref:`SOTA
-System<SOTA-system>`.
+System<getting-started-sota>`.
```

Comment:
The rst filename is getting-started-SOTA.rst
The reference has SOTA in small letters <getting-started-sota>. Does it work?

Reply:
Yes, I don't think it refers to the filename but a label